apps/namadillo/package.json

```diff
@@ -42,16 +42,17 @@
   },
   "scripts": {
     "bump": "yarn workspace namada run bump --target apps/namadillo",
+    "git:hashes": "VITE_GIT_SHORT=$(git rev-parse --short HEAD) VITE_GIT_LONG=$(git rev-parse HEAD)",
     "release": "release-it --verbose --ci",
     "release:dry-run": "release-it --verbose --dry-run --ci",
     "release:no-npm": "release-it --verbose --no-npm.publish --ci",
     "start:proxy": "node ./scripts/startProxies.js",
-    "dev": "vite",
+    "dev": "VITE_GIT_SHORT=$(git rev-parse --short HEAD) VITE_GIT_LONG=$(git rev-parse HEAD) vite",
     "preview": "vite preview",
     "dev:local": "NODE_ENV=development NAMADA_INTERFACE_LOCAL=\"true\" yarn dev",
     "dev:proxy": "VITE_PROXY=true && ./scripts/start-proxies.sh && yarn dev:local",
-    "build": "NODE_ENV=production && yarn wasm:build && vite build",
-    "build:only": "NODE_ENV=production &&  vite build",
+    "build": "NODE_ENV=production && yarn wasm:build && yarn git:hashes && VITE_GIT_SHORT=$(git rev-parse --short HEAD) VITE_GIT_LONG=$(git rev-parse HEAD) vite build",
```

Comment:
can't we use only `yarn git:hashes && vite` here?

Reply:
Unfortunately, I couldn't get the envs to pass from one script to another, so I simplified it to one var and included where necessary. I guess this is fine for now /shrug